src/GHC/RTS/EventTypes.hs

```diff
@@ -418,6 +418,16 @@ data EventInfo
                        , heapProfResidency :: !Word64
                        , heapProfLabel :: !Text
                        }
+
+  | ProfSampleCostCentre
+                       { profCapset :: Capset
```

Comment:
Is there a reason not to make this strict?

Reply:
I made it strict now.

Reply:
Thanks!